src/iuse_actor.cpp

```diff
@@ -1519,6 +1553,21 @@ void firestarter_actor::resolve_firestarter_use( Character *p, map *here,
     }
 }
 
+bool firestarter_actor::resolve_start( Character *p, map *here,
+                                       const tripoint_bub_ms &pos, start_type type )
+{
+    switch( type ) {
+        case ST_FIRE:
+            return here->add_field( pos, fd_fire, 1, 10_minutes );
+        case ST_SMOKER:
+            return iexamine::smoker_fire( *p, pos );
+        case ST_KILN:
```

Comment:
Add a `debugmsg` for TODO stuff like this, so that if it accidentally slips through into the live game, we would at least get a bug report from the users.